setup.cfg

```diff
@@ -23,7 +23,7 @@ packages =
     lavalink
 python_requires = >=3.5.3
 install_requires =
-    websockets>=6.0,<8.0
+    websockets>=6.0,<7.0
```

Comment:
Before I approve this, can I get a reasoning for clamping this down? I'm not seeing any reason to further restrict this here.

Reply:
This is to match with d.py websockets requirements.